radar/dev/flatfileportal.yaml

```diff
@@ -0,0 +1,18 @@
+name: Flatfile Portal
+blip:
+  - date: 2020-06-26
+    ring: ASSESS
+description: |
+  An add-on for web-based applications to allow customers or internal users to upload spreadsheets of data.
+  It pairs with SaaS dashboard and AI-assisted field match feature.
+rationale: |
+  HiiRetail needs reach UI capabilities, easy to embed and use, to allow users upload csv/xls data,
```

Comment:
Think this is a typo should be "rich" instead of "reach"

Reply:
You're right